src/model/docker.ts

```diff
@@ -34,7 +34,11 @@ const Docker = {
     let runCommand = '';
 
     if (parameters.unityLicensingServer !== '') {
-      LicensingServerSetup.Setup(parameters.unityLicensingServer, parameters.actionFolder);
+      LicensingServerSetup.Setup(
+        parameters.unityLicensingServer,
+        parameters.actionFolder,
+        parameters.unityLicensingProductIds,
+      );
```

Comment:
I'm not sure about this suggestion as I don't have much details about `unityLicensingProductIds`'s actual format, but it sounds a like a good idea to do input validation.

Now we're not doing validation for other parameters so this might be out of scope. Feel free to discard.